docs/features/software-catalog/configuration.md

```diff
@@ -79,6 +79,38 @@ silently at startup for convenience. So you only have to list it if you want to
 supply a token for it - and if you do, you can also leave out the `apiBaseUrl`
 and `rawBaseUrl` fields.
 
+### Processor: github-discovery
+
+There is a special GitHub discovery processor for discovering catalog entities
+within a GitHub organization. The processor will crawl a given GitHub
+organization and register entities matching the configured path. This can be
+useful as an alternative to static locations or manually adding things to the
+catalog.
+
+To use the discovery processor, you'll need a GitHub integration set up as above
+with a `GITHUB_TOKEN`. Then you can add a location target to the catalog
+configuration:
+
+```yaml
+catalog:
+  locations:
+    - type: github-discovery
+      target: https://github.com/myorg/service-*/blob/main/catalog-info.yaml
+```
+
+Note the `github-discovery` type, as this is not a regular `url` processor.
+
+The target is composed of three parts:
+
+- The base organization URL, `https://github.com/myorg` in this case
+- The repository blob to scan, which accepts \* wildcard tokens. This can simply
+  be `*` to scan all repositories in the organization. This example only looks
+  for repositories prefixed with `service-`.
+- The path within each repository to find the catalog YAML file. This will
+  usually be `/blob/main/catalog-info.yaml`, `/blob/master/catalog-info.yaml` or
+  a similar variation for catalog files stored in the root directory of each
+  repository.
+
```

Comment:
Hey how would you feel about having this (and other integration specific plugins) under the integration section of the docs instead? Could that make sense?

There's a definitely a bit of a fuzzy line between what should go where. But maybe this file could be more about specifically setting up the catalog as a product, and then the integrations could be about how to wire up specific things with it

Reply:
Makes a lot of sense, I think. As a first-time user I wanted a "configure all the Github things" section but it's sprinkled around right now. Your comment makes me want to update this page as just:

- Static location configuration (simplest getting-started path)
- Catalog rules
- Processors (discussing at a high-level but without code examples per se, links out to integrations)
- Link to `External integrations` as an advanced topic

Then move the Github integration example config to Integrations/Github along with a page for github-discovery. Seem reasonable?

Reply:
Yep something like that! But you mentioned `github-discovery` both in a bullet point and in the paragraph about the integrations docs, maybe I'm misreading but those should probably go in the latter too righr